CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+<!-- markdownlint-disable -->
```

Comment:
No issue with this change, but I'm curious if you can instead put this in a `.markdownlintignore` or something? Basically, my personal default (using `npx create-rwjblue-release-it-setup`) assumes that it can always _prepend_ the new changelog entries to the `CHANGELOG.md` file and having this at the top would kinda break that workflow.

Reply:
See https://github.com/rwjblue/create-rwjblue-release-it-setup/blob/master/RELEASE.md for a rough description of my normal workflow (and maybe it will work for us here too 😸, its just a `npx create-rwjblue-release-it-setup` away 🙃).

Reply:
Thanks! I switched to using `.markdownlintignore` which I agree is a much better solution.

Also interested to give your release-it setup a try.